.jscsrc

```diff
@@ -2,11 +2,20 @@
   "esnext": true,
   "validateQuoteMarks": "'",
   "requireDotNotation": true,
-  "requireCurlyBraces": true,
   "requireCommaBeforeLineBreak": true,
   "requireOperatorBeforeLineBreak": true,
   "disallowTrailingWhitespace": true,
   "disallowMultipleLineStrings": true,
   "disallowMixedSpacesAndTabs": true,
-  "disallowEmptyBlocks": true
-}
+  "disallowEmptyBlocks": true,
+
```

Comment:
Why don't we just take one of the reasonable presets

https://github.com/jscs-dev/node-jscs/tree/master/presets